common/static/common/js/spec/discussion/view/discussion_thread_list_view_spec.js

```diff
@@ -224,6 +224,19 @@
         };
 
         describe('should filter correctly', function() {
+            it('when no post message exists when alternate term was searched', function() {
```

Comment:
I think we should write this test in a separate spec suit. It doesn't fit in `should filter correctly` describe function. 
The string of new spec suit can be like `change filter should clear search alert`.

Reply:
Okay @Rabia23  I will change it.